File: ddsk.py
import random

DDSK = ["ドド","スコ"]
End = "ドドスコスコスコ" * 3
judge = ""
n = 0

while(True):
    t = random.randint(0,1) % 2
    print(DDSK[t])
    judge += DDSK[t]

    if len(judge) >= 24:
        if judge[2*n:] == End:
            break
        n += 1

print("ラブ注入❤︎")

File: ddsk.js
const DDSK = ["ドド","スコ"]
let result = 0

while(0x777 != (result & 0x777)){
    let LSD = Math.floor(Math.random() * 2)
    result <<= 1
    result = LSD|(result&~1)
    console.log(DDSK[LSD])
}

console.log("ラブ注入❤︎")
